Library/ViewModels/ManagePledgePaymentMethodViewModel.swift

```diff
@@ -0,0 +1,100 @@
+import KsApi
+import Prelude
+import ReactiveExtensions
+import ReactiveSwift
+import UIKit
```

Comment:
Can we get rid of the `UIKit` import here and just return the `imageName` as a string? I think we generally try to keep our VMs decoupled from UIKit.